modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -764,25 +764,15 @@ private ObjectDefinition _addObjectDefinition(
 		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-154872"))) {
 			_objectFieldLocalService.addSystemObjectField(
 				userId, objectDefinition.getObjectDefinitionId(), "Date",
-				ObjectEntryTable.INSTANCE.status.getName(),
-				ObjectEntryTable.INSTANCE.getTableName(), "Date", false, false,
-				null,
+				ObjectEntryTable.INSTANCE.createDate.getName(), "ObjectEntry",
```

Comment:
I think we forgot to fix the dbTableName here (if systemObjectField, dbTableName=Instance Table; If customObjectField, dbTableName=ObjectEntry)